src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const MIN_DAYS_FOR_DISCOUNT = 3;
+  const LONG_TERM = 7;
+  const PRICE = 40;
+  let sale = 20;
+
+  if (days >= MIN_DAYS_FOR_DISCOUNT && days < LONG_TERM) {
+    return days * PRICE - sale;
+  }
```

Comment:
u can do the `Long` condition first, then u won't have to do 2 checks here)
That is, first we check for a larger number

```suggestion
  if (days >= LONG_TERM) {
    return days * PRICE - sale;
  }
```